_members/pos-03-latinx-community-liaison.md

```diff
@@ -0,0 +1,6 @@
+---
+name: 'Open Position'
+title: 'Latinx Community Liaison'
```

Comment:
A new position - I'd like to "hire" for.

I'd like to add job descriptions and openings next, so this will link to something meaningful.